src/main/java/tc/oc/pgm/api/setting/SettingValue.java

```diff
@@ -18,7 +18,9 @@
   CHAT_GLOBAL("chat", "global"), // Send to all players in the same match
   CHAT_ADMIN("chat", "admin"), // Send to all server operators
   DEATH_OWN("death", "own"), // Only send death messages involving self
-  DEATH_ALL("death", "all"); // Send all death messages, highlight your own
+  DEATH_ALL("death", "all"), // Send all death messages, highlight your own
+  PICKER_DISPLAY("picker", "display"), // Display the picker GUI when clicked
```

Comment:
`show` maybe?

Reply:
Thanks for the suggestion. That’s more user friendly 😄